docs/source/spec/core.rst

```diff
@@ -4045,6 +4045,112 @@ settings from a service.
             }
         }
 
+The values for ``outputToken`` and ``items`` are paths. Paths are a series of
```

Comment:
Minor: can you put italics on "Path" to show it's defining a term?

Reply:
Should use the `:dfn:` directive.